extensions/amp-analytics/0.1/events.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
```

Comment:
nit: 2017 (should only be the year that the file was created (not sure if I asked you to do this, but sorry if i did!))